Koha/Plugin/EDS/bootstrap/modules/pfi-results.tt

```diff
@@ -412,7 +412,7 @@
 [% IF ( OpacStarRatings == 'all' || Koha.Preference('Babeltheque') ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.rating.js"></script>[% END %]
 [% IF ( OverDriveEnabled ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/js/overdrive.js"></script>[% END %]
 <script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.checkboxes.min.js"></script>
-[% IF ( OpacHighlightedWords ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.highlight-3.js"></script>
+[% IF ( OpacHighlightedWords ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.highlight-5.js"></script>
```

Comment:
same as eds-detail.tt comment

Reply:
Fixed in most recent commit: [68f871c](https://github.com/ebsco/edsapi-koha-plugin/pull/138/commits/68f871c213c34349431975a391e1d45d1c0dce78)
Can you please reinstall/retest? Thank you.